packages/backend/src/apps/frappe/common/add-auth-header.js

```diff
@@ -0,0 +1,15 @@
+import { OAUTH_ENDPOINTS } from "./constants.js";
+
+const addAuthHeader = ($, requestConfig) => {
+  if (requestConfig.url.includes(OAUTH_ENDPOINTS.GET_TOKEN)) {
```

Comment:
You can use additionalProperties for the requests.
```
if (requestConfig.additionalProperties?.skipAddingAuthHeader)
    return requestConfig;
```
example usage: `/workspace/packages/backend/src/apps/gmail/auth/refresh-token.js`

Reply:
Done.